component-tests/kafka/publisher.py

```diff
@@ -0,0 +1,37 @@
+""" Executed against a kafka instance in CI."""
+import time
+
+from pubsubs.registry import Registry
+
+# Two minutes from now
+TIMEOUT = time.time() + 60 * 2
+CONFIG = """\
+pubsubs:
+    myKafka:
+        backend: kafka
+        listeners: ['kafka:9092']
+        publisher:
+            poll: 1.0
+            message.timeout.ms: 3000
+        subscriber:
+            poll: 0.8
+            group.id: mygroup
+            auto.offset.reset: earliest
+"""
+
+registry = Registry()
+registry.register_from_config(CONFIG)
+
+kafka = registry["myKafka"]
+kafka.publish(topic="mytopic", message="howl")
```

Comment:
As dicussed - prove connect not recalled on another publish and definitely publish more than 1 message (like 10 or 100 - 100 should be published and consumed in < 1s with no sleeps)